edx_proctoring/templates/proctored_exam/failed-prerequisites.html

```diff
@@ -26,23 +26,26 @@
     {% endfor %}
     </ol>
   </p>
-  <p>
+
+  {% if allow_unproctored %}
+    <p>
     {% blocktrans %}
     Due to unsatisfied prerequisites, you can only take this exam without proctoring.
     {% endblocktrans %}
-  </p>
+    </p>
 
-  <button class="gated-sequence start-timed-exam" data-attempt-proctored=false>
+    <button class="gated-sequence start-timed-exam" data-attempt-proctored=false>
```

Comment:
It feels like this should be pulled out as a separate included template.  The redundancy leads to issues like the language needlessly differing between: `I understand that I am not eligible for academic credit.` and `I am not interested in academic credit.`.

I could imagine the `Due to unsatisfied prerequisites...` comment remaining here in a separate if, or moving into the shared template with a conditional.

You may not like this idea, so feel free to discuss before diving in.